### ISSUE TITLE
[BUG] TailwindCSS classes not working with Vite preact-ts Template, but working with Vite react-ts Template

### ISSUE THREAD

**Issue by angelhdzmultimedia** [preactjs/preset-vite#42]
- [X] Check if updating to the latest Preact version resolves the issue

**Describe the bug**
I found out about preact via Vite. So read about its performance, so decided to try it instead of react.
Did the same steps I would normally do to install TailwindCSS.
TailwindCSS classes did not work.  Tried updating all the dependencies to latest versions. 

Immediately after, created a react Vite project. Same steps. TailwindCSS classes worked right away.

**To Reproduce**

If possible, please provide a CodeSandbox/Codepen that demonstrates the issue. You can use the following template: https://codesandbox.io/s/preact-x-preact-cli-3-starter-vj285y2rn3

3. See error

**Expected behavior**
What should have happened when following the steps above?

**Comment by marvinhagemeister** [comment]
Can confirm that tailwindcss works as expected for me with `vite`. Used a similar setup to what @JoviDeCroock posted.

**Comment by angelhdzmultimedia** [comment]
> Can confirm that tailwindcss works as expected for me with `vite`. Used a similar setup to what @JoviDeCroock posted.
> 
> @angelhdzmultimedia Curious if we're missing something. Can you specify in more detail what is not working for you?

Thnaks for being kind, for not implying I am dumb,  and for your time replying. Highly appreciated.

As I mentioned, I did the SAME steps in both templates (Vite preact and Vite react). I did a lot of extra steps to make the CSS classes work with preact. I cleared Chrome cache, I restarted Vite couple of times, uninstalled tailwind and resinstalled, reloaded VSCode, updated all dependencies using @latest, and moved the import './index.css' to App.vue.

I had a simple <span> with some tailwind classes like text-red-500, font-bold, text-3xl. The text remained black, default size, and regular weight.

So I deleted the preact project, created a new one using Vite react template. Did the same steps. And boom, red text, big text, bold text right away. 

I'm gonna give preact a second opportunity tonight to see what was wrong. Maybe it was a npm issue where tailwind failed to get installed? It had happened to me before, for example with Pinia. I posted in Discord and in Pinia's discusion page my issue with using "this" in a store action. I couldn't understand why I was getting this random issue, as I was following the guide religiously. At the end, it was npm (or yarn) who failed to install Pinia.

Wishing you success and a lot of health. Cheers.
